app/controllers/api/orgs_controller.rb

```diff
@@ -10,8 +10,12 @@ def show
     end
 
     def partners
-      @product = Product.find_by!(slug: params[:org_id])
-      render json: @product, serializer: ProductApiSerializer
+      @product = Product.find_by(slug: params[:org_id])
+      if !@product
+        @product = Product.find_by(id: params[:org_id])
+      end
+      @users = TransactionLogEntry.product_partners_with_balances(@product.id).sort_by{|a, b| -b}.map{|a, b| {user: UserApiSerializer.new(User.find(a)), coins: b}}
```

Comment:
Unused block argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used.<br>Line is too long. [163/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.